client/app/scripts/liveblog-syndication/actions/incoming-syndication.js

```diff
@@ -27,6 +27,23 @@ liveblogSyndication
                             });
                         });
                     });
+                },
+                publish: function(post) {
+                    postsService.savePost(post.blog, post, undefined, {post_status: 'open'})
+                        .then(function(post) {
+                            Dispatcher.dispatch({
+                                type: 'ON_YANKED_POST',
```

Comment:
To find out what 'yanked' means here (a word whose meaning isn't clear to me, a native speaker), I have to read the code. It means saved or removed. So, please change it to ON_SAVED_POST and ON_REMOVED_POST. You then just to have one more line of code, an extra case in the switch statement.